test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerConfigFactoryTests.cs

```diff
@@ -153,6 +153,7 @@ public void DefaultWorkerConfigs_Overrides_DefaultWorkerRuntimeVersion_AppSettin
             var config = configBuilder.Build();
             var scriptSettingsManager = new ScriptSettingsManager(config);
             var testLogger = new TestLogger("test");
+            _testEnvironment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1");
```

Comment:
Still want to make sure we've reviewed the test behavior here to ensure this won't invalidate the test.

Reply:
This test is primarily checking the worker configs are populated with right values, specifically the worker runtime verion. In that context, the behavior should be same for both placeholder mode and non-placeholder mode. By setting this env variable to the test, I effectively changed the test from "non-placeholder" mode test to "placeholder" mode test.

As discussed, offline, ideally, we should create 2 tests, one for placeholder and one for non-placeholder scenario. That is beyond the scope of this PR and can be addressed in future.